### ISSUE TITLE
Contributors are not shown on the official website; errors on CI are ignored.

### ISSUE THREAD

**Issue by okathira** [markuplint/markuplint#2674]
- 現時点のdevブランチ

## Describe the bug

現在、markuplint公式サイトの「コミュニティ」ページにある[コントリビューターのリスト](https://markuplint.dev/ja/community#%E3%82%B3%E3%83%B3%E3%83%88%E3%83%AA%E3%83%93%E3%83%A5%E3%83%BC%E3%82%BF%E3%83%BC)が表示されていません。[英語版ページ](https://markuplint.dev/community#contributors)も同様です。

[CIの"Website Building Test"](https://github.com/markuplint/markuplint/blob/dev/.github/workflows/build-site.yml) を動かして該当する箇所を確認すると、エラーが出ている一方でCI自体はパスしてしまっていることがわかります。

```sh
npm warn exec The following package was not found and will be installed: prettier@3.5.3
Error:  Invalid configuration for file "/home/runner/work/markuplint/markuplint/website/src/components/Avatar/index.tsx":
Error:  Cannot find package '@markuplint-dev/prettier-config' imported from /home/runner/work/markuplint/markuplint/.prettierrc.mjs
error Command failed with exit code 2.
```

<details>

<summary>CIのログ 該当箇所全文</summary>

```sh
Run yarn site:build
  yarn site:build
  shell: /usr/bin/bash -e {0}
yarn run v1.[2](https://github.com/okathira/markuplint/actions/runs/14669452004/job/41172335404?pr=10#step:4:2)2.22
$ cd website; yarn; yarn site:build
[1/5] Validating package.json...
[2/5] Resolving packages...
[[3](https://github.com/okathira/markuplint/actions/runs/14669452004/job/41172335404?pr=10#step:4:3)/5] Fetching packages...
warning Pattern ["react-helmet-async@npm:@slorber/react-helmet-async@*"] is trying to unpack in the same destination "/home/runner/.cache/yarn/v6/npm-react-helmet-async-1.3.0-11fbc609[4](https://github.com/okathira/markuplint/actions/runs/14669452004/job/41172335404?pr=10#step:4:5)605cf60aa04a28c17e0aab894b4ecff-integrity/node_modules/react-helmet-async" as pattern ["react-helmet-async@npm:@slorber/react-helmet-async@1.3.0"]. This could result in non-deterministic behavior, skipping.
warning " > @docusaurus/core@3.7.0" has unmet peer dependency "@mdx-js/react@^3.0.0".
[4/[5](https://github.com/okathira/markuplint/actions/runs/14669452004/job/41172335404?pr=10#step:4:6)] Linking dependencies...
warning "@docusaurus/core > react-loadable-ssr-addon-v5-slorber@1.0.1" has unmet peer dependency "react-loadable@*".
warning "@docusaurus/core > @docusaurus/mdx-loader > @mdx-js/mdx > recma-jsx > acorn-jsx@5.3.2" has unmet peer dependency "acorn@^[6](https://github.com/okathira/markuplint/actions/runs/14669452004/job/41172335404?pr=10#step:4:7).0.0 || ^7.0.0 || ^8.0.0".
warning "@docusaurus/preset-classic > @docusaurus/plugin-debug > react-json-view-lite@1.5.0" has incorrect peer dependency "react@^16.13.1 || ^1[7](https://github.com/okathira/markuplint/actions/runs/14669452004/job/41172335404?pr=10#step:4:8).0.0 || ^18.0.0".
warning "@docusaurus/preset-classic > @docusaurus/theme-classic > @mdx-js/react@3.1.0" has unmet peer dependency "@types/react@>=16".
warning "@docusaurus/preset-classic > @docusaurus/theme-search-algolia > @docsearch/react > @algolia/autocomplete-preset-algolia@1.17.9" has unmet peer dependency "@algolia/client-search@>= 4.9.1 < 6".
warning "@docusaurus/preset-classic > @docusaurus/theme-search-algolia > @docsearch/react > @algolia/autocomplete-core > @algolia/autocomplete-plugin-algolia-insights@1.17.9" has unmet peer dependency "search-insights@>= 1 < 3".
warning "@docusaurus/preset-classic > @docusaurus/theme-search-algolia > @docsearch/react > @algolia/autocomplete-core > @algolia/autocomplete-shared@1.17.9" has unmet peer dependency "@algolia/client-search@>= 4.9.1 < 6".
warning " > eslint-plugin-react@7.37.5" has unmet peer dependency "eslint@^3 || ^4 || ^5 || ^6 || ^7 || ^[8](https://github.com/okathira/markuplint/actions/runs/14669452004/job/41172335404?pr=10#step:4:9) || ^9.7".
warning " > eslint-plugin-react-hooks@5.2.0" has unmet peer dependency "eslint@^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0-0 || ^[9](https://github.com/okathira/markuplint/actions/runs/14669452004/job/41172335404?pr=10#step:4:10).0.0".
[5/5] Building fresh packages...
$ yarn site:prebuild; docusaurus build
$ tsx scripts/prebuild/index.mts; npx prettier --write './src/**/*{ts,tsx,json,rc,md,mdx,css,scss}';
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
(Use `node --trace-warnings ...` to show where the warning was created)
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/attr-duplication.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/attr-duplication.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/attr-value-quotes.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/attr-value-quotes.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/case-sensitive-attr-name.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/case-sensitive-attr-name.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/case-sensitive-tag-name.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/case-sensitive-tag-name.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/character-reference.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/character-reference.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/class-naming.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/class-naming.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/deprecated-attr.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/deprecated-attr.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/deprecated-element.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/deprecated-element.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/disallowed-element.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/disallowed-element.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/doctype.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/doctype.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/end-tag.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/end-tag.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/heading-levels.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/heading-levels.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/id-duplication.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/id-duplication.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/ineffective-attr.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/ineffective-attr.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/invalid-attr.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/invalid-attr.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/label-has-control.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/label-has-control.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/landmark-roles.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/landmark-roles.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/neighbor-popovers.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/neighbor-popovers.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/no-ambiguous-navigable-target-names.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/no-ambiguous-navigable-target-names.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/no-boolean-attr-value.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/no-boolean-attr-value.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/no-consecutive-br.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/no-consecutive-br.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/no-default-value.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/no-default-value.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/no-duplicate-dt.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/no-duplicate-dt.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/no-empty-palpable-content.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/no-empty-palpable-content.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/no-hard-code-id.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/no-hard-code-id.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/no-orphaned-end-tag.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/no-orphaned-end-tag.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/no-refer-to-non-existent-id.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/no-refer-to-non-existent-id.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/no-use-event-handler-attr.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/no-use-event-handler-attr.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/permitted-contents.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/permitted-contents.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/placeholder-label-option.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/placeholder-label-option.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/require-accessible-name.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/require-accessible-name.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/require-datetime.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/require-datetime.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/required-attr.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/required-attr.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/required-element.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/required-element.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/required-h1.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/required-h1.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/table-row-column-alignment.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/table-row-column-alignment.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/use-list.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/use-list.md
(node:2023) V8: file:///home/runner/work/markuplint/markuplint/website/scripts/prebuild/utils.mts:1 'assert' is deprecated in import statements and support will be removed in a future version; use 'with' instead
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/wai-aria.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/wai-aria.md
✨ /home/runner/work/markuplint/markuplint/website/docs/rules/index.md
✨ /home/runner/work/markuplint/markuplint/website/i18n/ja/docusaurus-plugin-content-docs/current/rules/index.md
npm warn exec The following package was not found and will be installed: prettier@3.5.3
Error:  Invalid configuration for file "/home/runner/work/markuplint/markuplint/website/src/components/Avatar/index.tsx":
Error:  Cannot find package '@markuplint-dev/prettier-config' imported from /home/runner/work/markuplint/markuplint/.prettierrc.mjs
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[INFO] Website will be built for all these locales: 
- en
- ja
[INFO] [en] Creating an optimized production build...
[info] [webpackbar] Compiling Client
[info] [webpackbar] Compiling Server
[success] [webpackbar] Server: Compiled successfully in 23.23s
[success] [webpackbar] Client: Compiled successfully in 33.02s
[SUCCESS] Generated static files in "build".
[INFO] [ja] Creating an optimized production build...
[info] [webpackbar] Compiling Client
[info] [webpackbar] Compiling Server
[success] [webpackbar] Server: Compiled successfully in 19.97s
[success] [webpackbar] Client: Compiled successfully in 29.52s
[SUCCESS] Generated static files in "build/ja".
[INFO] Use `npm run serve` command to test your build locally.
Done in [10](https://github.com/okathira/markuplint/actions/runs/14669452004/job/41172335404?pr=10#step:4:11)5.80s.
```

</details>

試しに、各コマンドの `;` を `&&` で繋いでみたところ、CIがエラーを無視してしまうのは治りました。
また、CIでも最初に`yarn`をするようにしてみたところ、エラーが解決したように見えました。

[動作確認のPR](https://github.com/okathira/markuplint/pull/10)

## Steps To Reproduce

https://github.com/markuplint/markuplint/blob/dev/.github/workflows/build-site.yml の

```
on:
    pull_request:
        branches:
            - dev
        paths:
            - 'website/**'
            - '!website/**/*.md'
```

を発火させるため、

1. websiteディレクトリ以下の.mdファイル以外を適当に編集する。コメントをつけるなど。
2. devブランチに向けてPRを作成する。
3. "Website Building Test / build (pull_request)"を確認する

## The current behavior

- Websiteのビルドでエラーが出ている。
- しかし、CIのチェックとしてはそれが無視されている。

## The expected behavior

- Websiteのビルドが成功する。
- もし、エラーがあった場合はCIのチェックをすり抜けないようにする。
